tests/unit/output/test_json_output.py

```diff
@@ -89,7 +89,7 @@ def test_jmespath_json_response(self):
     def test_zero_value_is_printed(self):
         # Even though the integer 0 is false-like, we still
         # should be printing it to stdout if a jmespath query
-        # evalutes to 0.
+        # evaluates to 0.
```

Comment:
It looks like this file inadvertently changed its mode from 0644 to 0755.  Could you change it back?

Reply:
@jamesls good catch, sorry about that, not sure how it happened. Fixed in f1fbba61. Will squash.